docs/guides/driver_manager.md

```diff
@@ -77,7 +77,7 @@ The best place to find detailed information on driver manager installation is al
     === "conda"
 
         ```console
-        $ conda install -c conda-forge adbc-driver-manager
+        $ conda install -c conda-forge libadbc-driver-manager
```

Comment:
@amoeba I think:
https://anaconda.org/conda-forge/adbc-driver-manager <-- this is the Python one
https://anaconda.org/conda-forge/libadbc-driver-manager <-- this is the C++ one
The former might work because it contains the C++ library, but I think we want the latter.